services/minespace-web/src/components/pages/Incidents/IncidentPage.js

```diff
@@ -76,7 +76,6 @@ const defaultProps = {
   formErrors: {},
 };
 
-export const POST_SUBMISSION_INCIDENT_STATUSES = ["WNS", "AFR", "FRS", "UNR", "INV", "MIU", "CLD"];
```

Comment:
This is an export that wasn't imported anywhere else, it was only referenced in the useEffect below. I thought it made more sense to check against something constant (our _pre_-submission statuses of DFT and no status) than to keep adding to this list.